lib/sipper/cli.ex

```diff
@@ -1,17 +1,18 @@
 defmodule Sipper.CLI do
   @config_file System.user_home! <> "/.sipper"
 
-  def main([]) do
+  def main(args) do
+    parse_config_file ++ args
+    |> parse_args
+    |> run
+  end
```

Comment:
How do you feel about changing this something like the below?

``` elixir
  def main(cli_args) do
    config_file_args ++ cli_args
    |> parse_args
    |> run
  end
```

Specifically, since we're dealing with args from two places, we could give both of them `*_args` names – on line 5, you don't care about parsing (that's an internal concern of that other function), you only care about what it gives you.